backend/game/views/game.py

```diff
@@ -87,15 +87,14 @@ def join(self, request, *args, **kwargs):
                 and serializer.validated_data.get("password") != game.password
             )
         ):
-            return Response(
-                serializer.data, status=status.HTTP_403_FORBIDDEN, headers=headers
-            )
+            return Response(status=status.HTTP_403_FORBIDDEN, headers=headers)
```

Comment:
Il me semble que par défaut DRF ajoute un payload avec un message d'erreur, on devrait faire la même chose par homogénéité je pense

Reply:
Bonne idée, j'ai ajouté des messages d'erreur selon ce qui cause la réponse 403